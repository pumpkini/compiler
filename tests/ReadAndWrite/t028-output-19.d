int main() {
    Print(1.1111111E-2);
    Print(1.e10);
}
