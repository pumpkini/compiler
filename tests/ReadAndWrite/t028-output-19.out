0.01111111
10000000000.00000000
